packages/console/src/assets/docs/guides/native-flutter/README.mdx

```diff
@@ -221,18 +207,19 @@ void signOut() async {
 In Logto SDK, you can use `logtoClient.isAuthenticated` to check the authentication status, if the
 user is signed in, the value will be `true`, otherwise, the value will be `false`.
 
-```dart
+```dart title="lib/main.dart"
  bool isAuthenticated = await logtoClient.isAuthenticated;
 ```
 
 </Step>
 
-<Step title="Checkpoint: Test your application" subtitle="1 step">
+<Step title="Checkpoint: Test your application">
 
-Now let's wrap up the implementation and test your application.
+<Checkpoint />
 
-<Code className="language-dart">
-    {`import 'package:logto_dart_sdk/logto_dart_sdk.dart';
+<Code className="language-dart" title="lib/main.dart">
+{`import 'package:flutter/material.dart';
+import 'package:logto_dart_sdk/logto_dart_sdk.dart';
```

Comment:
does it actually looks sequential? check the sign-in and sign-out then a direct code snippet without any instruction.

Reply:
Yes, the code is a summary of all the instructions above. Implement the demo app for sign-in and sign-out. Looks sequential.